backend/src/routes/v2/model/webhook/putWebhook.ts

```diff
@@ -0,0 +1,63 @@
+import bodyParser from 'body-parser'
+import { Request, Response } from 'express'
+import { z } from 'zod'
+
+import { WebhookEvent, WebhookInterface } from '../../../../models/v2/Webhook.js'
+import { registerPath, webhookInterfaceSchema } from '../../../../services/v2/specification.js'
+import { updateWebhook } from '../../../../services/v2/webhook.js'
+import { parse } from '../../../../utils/v2/validate.js'
+
+export const putWebhookSchema = z.object({
+  params: z.object({
+    modelId: z.string(),
+    webhookId: z.string(),
+  }),
+  body: z.object({
+    name: z.string(),
+
+    uri: z.string(),
+    token: z.string().optional(),
```

Comment:
Traditionally this is generated by the server.  If it's easy to change, I might suggest we change it to that?  Otherwise, happy to keep it provided by the client.